project/Makefile

```diff
@@ -51,8 +51,14 @@ cs-fix-xml:
 	done
 .PHONY: cs-fix-xml
 
-test:
-	phpunit -c phpunit.xml.dist --coverage-clover build/logs/clover.xml
+build:
+	mkdir $@
+
+build/xdebug-filter.php: phpunit.xml.dist build
```

Comment:
Don't we need a clean method for this file?

Reply:
If we didn't for `build/logs/clover.xml`, then we do not for that one. And we probably do not because `build` is gitignore'd . I can implement that if you still feel the need for it.